bitnami/kafka/templates/broker/statefulset.yaml

```
@@ -26,6 +26,9 @@ spec:
       app.kubernetes.io/part-of: kafka
   serviceName: {{ printf "%s-broker-headless" (include "common.names.fullname" .) | trunc 63 | trimSuffix "-" }}
   updateStrategy: {{- include "common.tplvalues.render" (dict "value" .Values.broker.updateStrategy "context" $ ) | nindent 4 }}
+  {{- if and .Values.broker.minReadySeconds (semverCompare ">= 1.23" (include "common.capabilities.kubeVersion" .)) }}
```

As mentioned in https://github.com/bitnami/charts/pull/15417, using the `>=1.23-0` version constraint allows us to includes pre-releases and follows the general pattern used in Bitnami Helm charts:

```suggestion
  {{- if and .Values.broker.minReadySeconds (semverCompare ">= 1.23-0" (include "common.capabilities.kubeVersion" .)) }}
```

---

Thanks; I completely missed that.